packages/task/src/node/task-server.ts

```diff
@@ -112,6 +126,31 @@ export class TaskServerImpl implements TaskServer, Disposable {
                             problems
                         });
                     }
+                    if (taskConfiguration.isBackground) {
+                        const backgroundTaskStatus = this.toBackground.get(event.taskId);
```

Comment:
maybe we could simply rename `this.toBackground` into `this.backgroundTaskStatusMap` ?

Reply:
fixed, done !